src/complex/Filter/Output.hpp

```diff
@@ -107,13 +107,30 @@ class COMPLEX_EXPORT IDataCreationAction : public IDataAction
   DataPath m_CreatedPath;
 };
 
+/**
+ * @brief
+ */
+struct COMPLEX_EXPORT DataModifiedAction
+{
+
+  enum class ModifiedType : uint64
+  {
+    DataArrayModified = 0,
+    DataArrayPossiblyDeleted = 1,
```

Comment:
```suggestion
    Modified = 0,
    PossiblyDeleted = 1,
```
We could apply this type to non DataArray objects.

Reply:
We could have more enums then? The fear is that you get high enough in the GUI and you lose access to the original object and all you have is the DataPath and the Enum. I would like to be able to keep with it some sort of piece of information that would further indicate _what_ was modified or _possibly_ deleted.

Reply:
If we want that information, then we could store `DataObject::Type` as well.

Reply:
ArrayModificationNotification

Reply:
DataObjectFutureModifications

Reply:
DataObjectExecuteModification